backend/user_profile_backend/controller/getUser.js

```diff
@@ -52,7 +52,8 @@ const loginUser = async (request, response) => {
             const user = results.rows[0]
             const isMatch = await bcrypt.compare(password, user.password)  // assuming the password column is named 'password'
             if (isMatch) {
-                const token = jwt.sign({ userId: user.id }, 'yourSecretKey', { expiresIn: '1h' })
+                delete user["password"]
+                const token = jwt.sign({ "userdata": user }, 'yourSecretKey', { expiresIn: '1h' })
```

Comment:
Can the signing be changed to:
jwt.sign( {"userId": user.id, "isAdmin" : user.is_admin}, ....) 

We only need those two fields

Reply:
Changes have been made!